### ISSUE TITLE
Progressive Paginator has 508 and WCAG2AA a11y violations at collapsed mobile view

### ISSUE THREAD

**Issue by dev-cprice** [cerner/terra-core#1635]
# Bug Report

## Description

There are 508 and WCAG2AA a11y violations for the Progressive Paginator at the collapsed mobile view.

![pp-508](https://user-images.githubusercontent.com/34115417/41789253-44ff71f8-7614-11e8-8427-0b0640abc3c6.png)
![pp-508-2](https://user-images.githubusercontent.com/34115417/41789256-46448c38-7614-11e8-8d5d-92f0e297df3c.png)
![pp-wcag2aa](https://user-images.githubusercontent.com/34115417/41789259-47dac648-7614-11e8-842d-838e7ac49d0d.png)
![pp-wcag2aa-2](https://user-images.githubusercontent.com/34115417/41789261-495be0a6-7614-11e8-8e3d-abe97c01eab9.png)

## Steps to Reproduce

1. Navigate to the progressive paginator on the [doc site](https://engineering.cerner.com/terra-core/#/components/terra-paginator/paginator/progressive-paginator)
2. Set the window size to small enough so that the progressive paginator looks like the following:

![screen shot 2018-06-22 at 12 10 24 pm](https://user-images.githubusercontent.com/34115417/41789538-439ddc86-7615-11e8-8157-afa1c16e8a03.png)

3. Open up the [HTML CodeSniffer](https://squizlabs.github.io/HTML_CodeSniffer/).
4. Set the code sniffer filter to Section 508, observe the two a11y errors. 
5. Set the code sniffer filter to WCAG2AA, observe the four a11y errors.

## Expected Behavior

No a11y violations.

## Environment

* Component Name and Version: terra-paginator 1.5.0
* Browser Name and Version: all browsers
* Node/npm Version: node v8.11.2, npm v6.1.0
* Webpack Version: 
* Operating System and version (desktop or mobile):
